sdk/data/azcosmos/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.3.4 (Unreleased)
 
 ### Features Added
+* Added `NewPartitionKeyNull` function to create and query documents with no partition key in CosmosDB
```

Comment:
Null partition key is for documents with `Null` in the Partition Key, not for documents without Partition Key.

There is another value (`None`) which is the one for that case.

If you are looking to add the scenario with no Partition Key, I think Null is not the right one

Reply:
Have edited the changelog. The use case is for null partition key